build/Targets/VSL.Imports.targets

```diff
@@ -204,6 +204,16 @@
     <ProjectLanguage Condition="'$(MSBuildProjectExtension)' == '.vbproj' OR '$(Language)' == 'VB'">VB</ProjectLanguage>
     <ProjectLanguage Condition="'$(MSBuildProjectExtension)' == '.vcxproj' OR '$(Language)' == 'C++'">C++</ProjectLanguage>
   </PropertyGroup>
+
+  <!-- VSIX projects generate a temp project file which doesnt have any extension and needs to be built as C# projects. The .NET SDK
+       just imporrts Microsoft.Common.CurrentVersion.targets. So set LanguageTargets to import CSharp.targets if the 
+       extension is unknown -->
```

Comment:
Yes, I believe this was pointed out on the bug thread as well. It would be great to know exactly what is missing and ensure it is in the SDK targets by default.